docs/attributes-registry/process.md

```diff
@@ -17,3 +17,21 @@
 | `process.parent_pid` | int | Parent Process identifier (PID). | `111` |
 | `process.pid` | int | Process identifier (PID). | `1234` |
 <!-- endsemconv -->
+
+## Process runtime attributes
```

Comment:
Do we need a new section here? We'll have separation by sections in the `resource/process.md` anyways.